README.md

```diff
@@ -250,7 +278,23 @@ will take precedence.
 
 Hyrax is available under [the Apache 2.0 license](LICENSE.md).
 
-# Contributing
+# Quick Start for Hyrax development and contributing code back
```

Comment:
@mjgiarlo RE: Reordering the sections... in my opinion is part of the fix for the PermissionsTemplate error.  If someone skips the Quick Start steps and just reads the doc top-down through the sections, they will still get the error.  I reordered the sections to make sure they are in the same order as the Quick Start.